generator/.DevConfigs/4f49cfc6-d4a8-477c-a685-6b96d52d73ab.json

```diff
@@ -0,0 +1,11 @@
+{
+    "services": [
+        {
+            "serviceName": "S3",
+            "type": "patch",
+            "changeLogMessages": [
+                "Split GetACL into GetBucketACL and GetObjectACL. Split PutACL into PutObjectACL and PutBucketACL. Obsolete PutACL and GetACL"
```

Comment:
I'd expand this message a bit more (for example, if this goes into the V4 tracker issue, is there enough information for customers to understand why we're doing this?)